iiif_prep/iiif_prep/preparer.py

```diff
@@ -0,0 +1,153 @@
+import logging
+
+from configparser import ConfigParser
+from os import listdir
+from os.path import isdir, join
+from shutil import copytree
+
+from .archivesspace import ArchivesSpaceClient, NoResultsError, MultipleResultsError
+
+
+class Preparer():
+    CURRENT = 0
+    MICROFILM = 1
+    LEGACY = 2
+
+    def __init__(self):
+        logging.basicConfig(
+            datefmt='%m/%d/%Y %I:%M:%S %p',
+            filename='iiif_preparer.log',
+            format='%(asctime)s %(message)s',
+            level=logging.INFO)
+        self.config = ConfigParser()
+        self.config.read("local_settings.cfg")
+
+    def run(self, source_directory, target_directory):
+        as_client = ArchivesSpaceClient(
+            self.config.get("ArchivesSpace", "baseurl"),
+            self.config.get("ArchivesSpace", "username"),
+            self.config.get("ArchivesSpace", "password"),
+            self.config.get("ArchivesSpace", "repository"))
```

Comment:
```suggestion
            self.config.get("ArchivesSpace", "password"))
```
I am pretty sure you don't need to pass the repo to authenticate, but rather the repository is an attribute on the `ASpace` object.